azure-pipelines/maven-release/common.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     gradleGeneratePomFiletask: common:generatePomFileForDistReleasePublication
     aarSourceFolder: common/build/outputs/aar
     jarSourceFolder: common/build/outputs/jar
-    pomSourceFolder: common/build/publications/distRelease
+    pomSourceFolder: common/build/poms
```

Comment:
is this path changed because of the new GenerateMavenPom task added in build.gradle?

Reply:
yes. I changed the path to publish under poms folder.